Note about Scheduled Workflows (#5467)
circleci/circleci-docs: jekyll/_cci2/dynamic-config.md

parent configuration file (in the `.circleci/` directory). This will designate that `config.yaml` as a `setup workflow` 
configuration, enabling you and your team to get up and running with dynamic configuration.

**Note**: *At this time, Dynamic Configuration does not work with Scheduled Workflows. We are working on

> Scheduled Pipelines, which will alleviate this. They are expected to launch in the near future. A possible work around can be found on our Discuss page: https://discuss.circleci.com/t/workaround-using-scheduled-workflows-with-dynamic-config/40344 *

Reviewer: ```suggestion
Scheduled Pipelines, which will alleviate this. They are expected to launch in the near future. A possible work around can be found on our [Discuss page](https://discuss.circleci.com/t/workaround-using-scheduled-workflows-with-dynamic-config/40344).
```

> Scheduled Pipelines, which will alleviate this. They are expected to launch in the near future. A possible work around can be found on our [Discuss page](https://discuss.circleci.com/t/workaround-using-scheduled-workflows-with-dynamic-config/40344).

Reviewer 2: ```suggestion
Scheduled Pipelines, which will alleviate this. This feature is expected to launch in the near future. A possible work around can be found on our [Discuss page](https://discuss.circleci.com/t/workaround-using-scheduled-workflows-with-dynamic-config/40344).
```
